.eslintrc.js

```diff
@@ -16,6 +16,8 @@ module.exports = {
     'ember/avoid-leaking-state-in-ember-objects': 'off',
     'ember/no-get': 'off', // @TODO remove support for old Ember
     'ember/no-classic-classes': 'off',
+    'simple-import-sort/exports': 'error',
+    'simple-import-sort/imports': 'error',
```

Comment:
Out-of-order import statements can be easily fixed by running `yarn lint:js:fix`.